src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLoggerOptions.cs

```diff
@@ -105,6 +105,9 @@ public string QuickPulseAuthenticationApiKey
         /// </summary>
         public bool EnableDependencyTracking { get; set; } = true;
 
+
+        public bool EnableSqlCommandTextInstrumentation { get; set; } = false;
```

Comment:
I think we should put this into a new  `DependencyTrackingOptions` type, which then lives here. It looks like there's a handful of other Enable/Disable properties on the module that we can also add to this type and just cover them all at once..